gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/test/java/io/gravitee/rest/api/management/v2/rest/mapper/PageMapperTest.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright © 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.rest.api.management.v2.rest.mapper;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.gravitee.rest.api.management.v2.rest.model.SourceConfiguration;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+/**
+ * @author Sergii ILLICHEVSKYI (sergii.illichevskyi at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+class PageMapperTest extends AbstractMapperTest {
+
+    private PageMapper pageMapper;
+
+    @BeforeEach
+    void setUp() {
+        pageMapper = PageMapper.INSTANCE;
+    }
+
+    @Test
+    @DisplayName("Should map SourceConfigurationConfiguration to JSON string")
+    void shouldMapSourceConfigurationConfigurationToJsonString() throws JsonProcessingException {
+        var sourceConfiguration = new SourceConfiguration();
+        sourceConfiguration.setType("Type1");
+        sourceConfiguration.setConfiguration("Configuration1");
+
+        String expected = """ 
+class SourceConfiguration {
```

Comment:
Shouldn't it be simply the properties + values of a class? It seems a bit weird that it would include the class name.

Reply:
No, this is how toString is generated in that class from the open API
<img width="699" alt="Screenshot 2024-07-01 at 16 59 09" src="https://github.com/gravitee-io/gravitee-api-management/assets/8080303/ddbc4281-3bd7-4b26-b4f7-393afc8bbc06">

Reply:
Ok, then it needs to be serialized differently than just `.toString()`. With just calling `.toString`, the `class SourceConfiguration {` would be saved in the database for the `configuration` field, which is not ok. Perhaps you can use another method? We do this with lots of other objects, including `Api`.

Reply:
Done